templates/widget-with-ui/manifest.json

```diff
@@ -2,7 +2,7 @@
   "name": "{{widgetName}}",
   "id": "{{widgetId}}",
   "api": "1.0.0",
-  "editorType": ["figjam"],
+  "editorType": [{{{widgetEditorType}}}],
```

Comment:
we need to use 3 sets of braces here to prevent mustache from automatically replacing `"` with `&quot;`